app/models/manageiq/providers/ansible_tower/inventory/collectors/automation_manager.rb

```diff
@@ -0,0 +1,19 @@
+class ManageIQ::Providers::AnsibleTower::Inventory::Collectors::AutomationManager < ManagerRefresh::Inventory::Collector
+  def inventories
+    manager.with_provider_connection do |connection|
+      connection.api.inventories.all
+    end
+  end
+
+  def hosts
+    manager.with_provider_connection do |connection|
+      connection.api.hosts.all
+    end
+  end
+
+  def job_templates
+    manager.with_provider_connection do |connection|
```

Comment:
yes, in aws I have methods holding the connection cached https://github.com/Ladas/manageiq-providers-amazon/blob/a32f6e82199d72c731e413c89d110e0afacbda82/app/models/manageiq/providers/amazon/inventory/collector.rb#L88-L88